### PR TITLE
statements GET and POST with prisma

### PR DESCRIPTION
**Pull request by eric-gt**
- change from MongoDB integration to Postgres via Prisma
- created development and testing .env files
- created schema.prisma with Statement Model
- edited testing environment to use Postgres and Prisma
- tests passing for basic GET and POST operations

### CODE REVIEW

File: pages/api/helpers/api/error-handler.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ function errorHandler (
   res: NextResponse
 ): void {
   if (typeof(err) === 'string') {
+    console.log(err);
```
Comment (calufa) [inline pages/api/helpers/api/error-handler.ts:10]: Could we use `console.error()`? https://www.w3schools.com/jsref/met_console_error.asp.

In k8s world, containers that wish to report an error should write to the error interface. In a managed k8s cluster like EKS, the cloud provider typically has a modified version of k8s' logs and error collection libraries and will redirect logs to their cloud log and offerings.

In the same fashion, using `console.log()` should redirect logs to k8s' log collection mechanism.

File: lib/db.ts [hunk 2]
```diff
@@ -1,9 +1,4 @@
...
-import { mongoDBURI } from "/app/config";
+import { PrismaClient } from "@prisma/client";
 
-const options = {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-};
-
-export default new MongoClient(mongoDBURI, options);
+let dbClient = new PrismaClient()
```
Comment (calufa) [inline lib/db.ts:3]: it is always seductive to use `let`, but unless it's _strictly_ necessary, I would always default to `const`.

File: pages/api/xAPI/statements/index.ts [hunk 3]
```diff
@@ -59,27 +61,20 @@ async function runMiddleware(
...
-      .toArray();
-    rows.forEach((row) => {
-      delete row["_id"]; //_id is a mongoDB parameter outside the scope of xAPI
-    });
+      .statement
+      .findMany();
+    const statements = rows.map((row) => row.statement);
     res.status(200).json({
-      statements: rows,
+      statements: statements,
```
Comment (calufa) [inline pages/api/xAPI/statements/index.ts:72]: Interestingly, in javascript/typescript world:

```
res.status(200).json({
      statements: statements,
```

and 

```
res.status(200).json({
      statements,
```

are equivalent.

Comment (calufa) [inline pages/api/xAPI/statements/index.ts:72]: Interestingly, in javascript/typescript world:

```
res.status(200).json({
      statements: statements,
```

and 

```
res.status(200).json({
      statements,
```

are equivalent.

**Comment by eric-gt** [comment]
Obsolete

**Pull request closed without merging**